### PR TITLE
Feature/get orders

### PR DESCRIPTION
**Pull request by samicodesit**
Updated getOrders function to match the latest API.

### CODE REVIEW

File: src/lib/products.js [hunk 1]
```diff
@@ -141,14 +141,31 @@ class Products extends Request {
...
+  } = {}) {
+    // URL query params
+    const queryParams = {
+      page,
+      limit,
+      status,
+      purchaseorderNumber
+    };
+
+    return this.$fetch('GET', '/orders', { queryParams });
```
Comment (HazemKhaled) [inline src/lib/products.js]: Codacy Issue found: [Strings must use doublequote.](https://app.codacy.com/manual/Knawat/knawat-suppliers-npm/pullRequest?prid=5119557)

**Comment by HazemKhaled** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Knawat/knawat-suppliers-npm/pullRequest?prid=5119557&bid=16832779)

**Comment by HazemKhaled** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Knawat/knawat-suppliers-npm/pullRequest?prid=5119557&bid=16832779)

File: README.md [hunk 2]
```diff
@@ -192,13 +192,21 @@ sa.getCategories();
...
+  status = null,
+  purchaseorderNumber = null
+}
 sa.getOrders(limit, page);
 ```
 
 | Params  | Type     | Description                                  |
 | ------- | -------- | -------------------------------------------- |
-| `limit` | `Number` | Number of orders to retrieve. `Default: 1`   |
+| `limit` | `Number` | Number of orders to retrieve. `Default: 20`   |
```
Comment (HazemKhaled) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/manual/Knawat/knawat-suppliers-npm/pullRequest?prid=5119557)

File: README.md [hunk 3]
```diff
@@ -192,13 +192,21 @@ sa.getCategories();
...
 | `page`  | `Number` | Number of the page to retrieve. `Default: 1` |
+| `status` | `String` | Status of orders to retrieve. `Allowed fields : open, issued, cancelled, billed, closed. Default: null`   |
```
Comment (HazemKhaled) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/manual/Knawat/knawat-suppliers-npm/pullRequest?prid=5119557)

File: README.md [hunk 4]
```diff
@@ -192,13 +192,21 @@ sa.getCategories();
...
+| `purchaseorderNumber`  | `String` | Purchase number of the order to retrieve. `Default: null` |
```
Comment (HazemKhaled) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/manual/Knawat/knawat-suppliers-npm/pullRequest?prid=5119557)

**Comment by HazemKhaled** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Knawat/knawat-suppliers-npm/pullRequest?prid=5119557&bid=16832779)

File: README.md [hunk 5]
```diff
@@ -192,13 +192,21 @@ sa.getCategories();
...
+  status = null,
+  purchaseorderNumber = null
+}
 sa.getOrders(limit, page);
 ```
 
 | Params  | Type     | Description                                  |
 | ------- | -------- | -------------------------------------------- |
-| `limit` | `Number` | Number of orders to retrieve. `Default: 1`   |
+| `limit` | `Number` | Number of orders to retrieve. `Default: 20` |
```
Comment (HazemKhaled) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/manual/Knawat/knawat-suppliers-npm/pullRequest?prid=5119557)

File: README.md [hunk 6]
```diff
@@ -192,13 +192,21 @@ sa.getCategories();
...
 | `page`  | `Number` | Number of the page to retrieve. `Default: 1` |
+| `status` | `String` | Status of orders to retrieve. `Allowed fields : open, issued, cancelled, billed, closed. Default: null` |
```
Comment (HazemKhaled) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/manual/Knawat/knawat-suppliers-npm/pullRequest?prid=5119557)

File: README.md [hunk 7]
```diff
@@ -192,13 +192,21 @@ sa.getCategories();
...
+| `purchaseorderNumber` | `String` | Purchase number of the order to retrieve. `Default: null` |
```
Comment (HazemKhaled) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/manual/Knawat/knawat-suppliers-npm/pullRequest?prid=5119557)

**Comment by ismail9k** [comment]
Duplicated.

**Pull request closed without merging**